static/js/actions/financial_aid.js

```diff
@@ -36,6 +36,8 @@ export const addFinancialAid = (income: number, currency: string, programId: num
     return api.addFinancialAid(income, currency, programId).then(
       () => {
         dispatch(receiveAddFinancialAidSuccess());
+        dispatch(fetchCoursePrices());
+        dispatch(fetchDashboard());
```

Comment:
this is a good idea overall, this way you can't forget to put the right stuff in `.then`!